backend/api/views/factories_cr.py

```diff
@@ -104,6 +104,7 @@ def _handle_create_ohshown_events(request):
         "name": post_body["name"],
         "lat": post_body["lat"],
         "lng": post_body["lng"],
+        "sight_see_date_time": post_body["datetime"],
```

Comment:
@yanghaochang104 this is where I found the issue https://github.com/OH-SHOWN/ohshown-frontend/issues/71

Reply:
I'll take a look at it.
Thanks for your report!

Reply:
@yanghaochang104 's investigation shows the issue is not caused by the FE https://github.com/OH-SHOWN/ohshown-frontend/issues/71 and is suspiciously caused by here https://github.com/OH-SHOWN/ohshown-backend/pull/59#discussion_r864888617

Reply:
Fixed by 8261ec36aaa6e67928211137fb0785391e00ac35 !